Documentation/cheatsheet.rst

```diff
@@ -2,157 +2,396 @@
+    Cilium health daemon:   Ok
+    Controller Status:      6/6 healthy
+    Proxy Status:           OK, ip 10.15.28.238, port-range 10000-20000
+    Cluster health:   1/1 reachable   (2018-04-11T07:33:09Z)
+    $
+
+Get a detailed status of the agent:
+::
+
+    $ cilium status --all-controllers --all-health --all-redirects
+    KVStore:                Ok         Consul: 172.17.0.3:8300
+    ContainerRuntime:       Ok
+    Kubernetes:             Disabled
+    Cilium:                 Ok         OK
+    NodeMonitor:            Listening for events on 2 CPUs with 64x4096 of shared memory
+    Cilium health daemon:   Ok
+    Controller Status:      6/6 healthy
+      Name                                 Last success   Last error   Count   Message
+      kvstore-lease-keepalive              2m52s ago      never        0       no error
+      ipcache-bpf-garbage-collection       2m50s ago      never        0       no error
+      resolve-identity-29898               2m50s ago      never        0       no error
+      sync-identity-to-k8s-pod (29898)     50s ago        never        0       no error
+      sync-IPv4-identity-mapping (29898)   2m49s ago      never        0       no error
+      sync-IPv6-identity-mapping (29898)   2m49s ago      never        0       no error
+    Proxy Status:   OK, ip 10.15.28.238, port-range 10000-20000
+    Cluster health:         1/1 reachable   (2018-04-11T07:32:09Z)
+      Name                  IP              Reachable   Endpoints reachable
+      runtime (localhost)   10.0.2.15       true        false
+    $
+
+Get the current agent configuration
+::
 
-	cilium policy import <my-policy.json>
+    cilium config
+
+Policy management
+-----------------
 
+
+Importing a policy
 ::
 
-	# Get list of all imported policy rules
+    cilium policy import my-policy.json
 
-	cilium policy get
 
+Get list of all imported policy rules
 ::
 
-	# Remove all policy
+	cilium policy get
+
+Remove all policy
+::
 
 	cilium policy delete --all
 
-Connectivity
-============
 
+Tracing
+~~~~~~~
+
+
+Check policy enforcement between two labels on port 80:
 ::
 
-	# Check cluster connectivity
+	cilium policy trace -s <app.from> -d <app.to> --dport 80
+
 
-	cilium-health status
+Check policy enforcement between two identities
+::
 
-Monitoring
-==========
+    cilium policy trace --src-identity <from-id> --dst-identity <to-id>
 
+Check policy enforcement between two pods:
 ::
 
-	# Monitor cilium datapath notifications
+    cilium policy trace --src-k8s-pod <namespace>:<pod.from> --dst-k8s-pod <namespace>:<pod.to>
+
+
+Monitoring
+~~~~~~~~~~~
 
-	cilium monitor 
 
+Monitor cilium datapath notifications
 ::
 
-	# Verbose output (including debug if enabled)
+    cilium monitor
 
-	cilium monitor -v
 
+Verbose output (including debug if enabled)
 ::
 
-        # Filter for only the events to endpoint
+    cilium monitor -v
 
-        cilium monitor --related-to=<id>
 
+Filter for only the events to endpoint
```

Comment:
s/events to endpoint/events related to endpoint

I would say that `events to endpoint` are handled by `--to` flag.